qiskit/providers/aer/openpulse/qobj/op_system.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable=invalid-name
+
+"The OpenPulse simulator system class"
+
+
+class OPSystem():
+    """ A Class that holds all the information
+    needed to simulate a given PULSE qobj
+    """
+    def __init__(self):
+        # The system Hamiltonian in numerical format
+        self.system = None
+        # The noise (if any) in numerical format
+        self.noise = None
+        # System variables
+        self.vars = None
+        # The initial state of the system
+        self.initial_state = None
+        # Channels in the Hamiltonian string
+        # these tell the order in which the channels
+        # are evaluated in the RHS solver.
+        self.channels = None
+        # options of the ODE solver
+        self.ode_options = None
+        # time between pulse sample points.
+        self.dt = None
```

Comment:
somewhere we need to make the simulator `dt` work with seconds, rather than nanoseconds.
because in terra this was recently changed: https://github.com/Qiskit/qiskit-terra/blob/37036f494d875fdb029b921043a0a8df4a4c2eb8/releasenotes/notes/extend-backend-configuration-129d7292471fe331.yaml#L19